data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,268 @@ event "remnant: pantica void sprites"
 	system "Nenia"
 		fleet "Ember Waste Node" 6000
+
+
+
+mission "Remnant: Expanded Horizons Astral 1"
+	name "Preparation for expedition"
+	description "Travel to <destination> to prepare for an expedition to Sagittarius A*. Ensure that you have at least six tons of outfit space available."
+	passengers 1
+	minor
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Quarg 3: done"
+		random < 25
+	on offer
+		conversation
+			`You glance into the starport cafeteria and see Dawn sitting at a table looking worried. She hasn't noticed you yet. Do you approach her?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+			`	She smiles when she sees you, and waves you to the seat opposite her. "Back from gallivanting around the galaxy with more tales to share?" she signs.`
+			choice
+				`	"Yes, as a matter of fact..."`
+				`	"No, just hunting Korath..."`
+				`	"No, just testing some stuff..."`
+				`	"You look worried. Is something happening?"`
+					goto worried
+			`	She listens for a minute or two as you tell her about what you have been up to, but it is obvious that she has something on her mind.`
+			choice
+				`	"You look worried. Is everything alright?"`
+				`	"You look distracted. Do you want to talk about it?"`
+			label worried
+			`	She makes a gesture of appreciation. "Yes, I'm worried, <first>. I keep feeling like there's something I'm missing." She frowns, contemplating something unseen. "I should back up a step. Since we last talked, I was assigned to serve with an astrophysics team for a year. My current work is part of the quantum condensate fabric density project, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine,  but I keep getting this prickly feeling that there's a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something obvious or repetitious really quickly. But the feeling just will not go away."`
+			choice
+				`	"Have you asked someone with more experience for advice?"`
+				`	"Don't worry about it."`
+					goto worry
+				`	"Can I help with something?"`
+					goto help
+			`	"I did. The senior researcher told me that most of the team feels that something is not quite right here. They say there are some potential theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
+				goto help
+			label worry
+			`	"Thanks for the advice, but that is our purpose here - to worry about all the stuff that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
+			label help
+			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport..."`
```

Comment:
Unpacking this a bit:
1) Yes, the Remnant do have JD, which is how they have sensor logs and recordings from previous visits.
2) She asks the player to assist because, well, the player is a known entity. She's already been on one research trip with the player, and that worked out nicely. 

What I was going for here is that she was in the processing of organizing an expedition, but hadn't arranged transportation yet. Since the player is here, and is a well known space traveler who has taken her on a previous research mission, then it makes sense to see if they are available to do so again.

The implication by the "and I could use a transport." isn't meant to imply that the *Remnant* need a transport, just that Dawn in particular does not have a JD equipped transport immediately available at her disposal.